ci/infra/testrunner/testrunner.py

```diff
@@ -70,11 +70,18 @@ def join_node(options):
 
 def join_nodes(options):
     skuba = Skuba(options.conf, options.platform)
-    skuba.join_nodes(
-        masters=options.masters,
-        workers=options.workers,
-        timeout=options.timeout
-    )
+    try:
+        skuba.join_nodes(
+            masters=options.masters,
+            workers=options.workers,
+            timeout=options.timeout
+        )
+    except Exception:
+        skuba.join_nodes(
```

Comment:
Not sure what this change has to do with the purpose of this PR. But in any case, looks like in case of an exception joining nodes, you are forcing to execute it again? Am I missing or misreading this change?

Reply:
Sorry, this is a mistake on my part. This is not passing the vmware tests once it passed them once, so I already removed the WIP tag but I shouldnt as I been using this PR as test bed for some WIP actions to remediate vmware issues.